src/components/SendTransaction/CreateTransaction.tsx

```diff
@@ -29,10 +33,15 @@ interface CreateProps {
   formData: FormData;
 }
 
+const isFederationAddress = (value: string) => value.includes("*");
```

Comment:
I don't love this check, but I can't think of a better one. As long as the form's sufficiently validated, this should be fine.

Reply:
Ya I took that from how Mobile Wallet checks. I assumed if it's used there it should be valid. @quietbits any objections to using this?